.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: csharp
 dist: trusty
 mono: none
-dotnet: 1.0.1
+dotnet: 1.0.4
```

Comment:
Is 1.0.4 the highest version we can move to? As I see it, we should bump the version to the highest possible supported one.

Reply:
updated to latest .NET core SDK